jsonrpc/src/content/results.json

```diff
@@ -90,6 +90,13 @@
       "$ref": "#/components/schemas/Enr"
     }
   },
+  "ValidateContentResult": {
+    "name": "validateContentResult",
+    "description": "Returns true or false depending on if the client could validate the given data",
```

Comment:
We should probably allow here to raise error with more information, to see what went wrong?
e.g. invalid parameters, content required to validate not found, or an actual validation failure. Perhaps in the latter case we can indeed just return false.

Reply:
Yeah I think that would definitly be the play. So we could return true (or maybe a enum if we support tiers of validation or something in the future idk) then just error on failure I think that is fine